### ISSUE TITLE
[FR] Add site names, handle and site groups to GraphQL

### ISSUE THREAD

**Issue by rtrudel** [Dunkel-Design/craft-graphql-sites#1]
We can currently retrieve the language, the baseUrl, and the id, but similarly, it would be interesting to have the name, the handle and the information regarding the site group, just like `craft.app.sites.getAllSites()` does.

If it is too complex for the site groups, at least the name and handle of the site in addition to the information already returned.

By the way, I don't understand why this isn't built into Craft CMS, it seems like that would be the normal place for it, so thank you for addressing this gap.

**Comment by soerenmeier** [comment]
I think you should now be able to get almost all fields.
Let me know if one is still missing or the group query is not what you need.

```gql
query {
  # csites for craft sites, because craft might wan't to use sites in the future
  csites {
    id
    baseUrl
    language
    name
    handle
    enabled
    hasUrls
    primary
    sortOrder
    group {
      id
      name
    }
  }
}
```

Is Published as v0.1.5

**Comment by rtrudel** [comment]
This is perfect! This is exactly what should be! Thank you!
